[zookeeper] build failure

### Operating system

Windows

### Compiler

MSVC

### Steps to reproduce the behavior

```Shell
.\vcpkg.exe install zookeeper
```

### Failure logs

Computing installation plan...
The following packages will be built and installed:
    zookeeper:x64-windows@3.8.4
Detecting compiler hash for triplet x64-windows...
Compiler found: C:/Program Files/Microsoft Visual Studio/2022/Enterprise/VC/Tools/MSVC/14.44.35207/bin/Hostx64/x64/cl.exe
Restored 0 package(s) from C:\Users\Firelight\AppData\Local\vcpkg\archives in 62.7 us. Use --debug to see more details.
Installing 1/1 zookeeper:x64-windows@3.8.4...
Building zookeeper:x64-windows@3.8.4...
-- Note: zookeeper only supports static library linkage. Building static library.
Downloading https://dlcdn.apache.org/zookeeper/stable/apache-zookeeper-3.8.4.tar.gz -> apache-zookeeper-3.8.4.tar.gz
error: https://dlcdn.apache.org/zookeeper/stable/apache-zookeeper-3.8.4.tar.gz: failed: status code 404
note: If you are using a proxy, please ensure your proxy settings are correct.
Possible causes are:
1. You are actually using an HTTP proxy, but setting HTTPS_PROXY variable to `https://address:port`.
This is not correct, because `https://` prefix claims the proxy is an HTTPS proxy, while your proxy (v2ray, shadowsocksr, etc...) is an HTTP proxy.
Try setting `http://address:port` to both HTTP_PROXY and HTTPS_PROXY instead.
2. If you are using Windows, vcpkg will automatically use your Windows IE Proxy Settings set by your proxy software. See: https://github.com/microsoft/vcpkg-tool/pull/77
The value set by your proxy might be wrong, or have same `https://` prefix issue.
3. Your proxy's remote server is out of service.
If you believe this is not a temporary download server failure and vcpkg needs to be changed to download this file from a different location, please submit an issue to https://github.com/Microsoft/vcpkg/issues
CMake Error at scripts/cmake/vcpkg_download_distfile.cmake:136 (message):
  Download failed, halting portfile.
Call Stack (most recent call first):
  ports/zookeeper/portfile.cmake:5 (vcpkg_download_distfile)
  scripts/ports.cmake:206 (include)


error: building zookeeper:x64-windows failed with: BUILD_FAILED
See https://learn.microsoft.com/vcpkg/troubleshoot/build-failures?WT.mc_id=vcpkg_inproduct_cli for more information.
Elapsed time to handle zookeeper:x64-windows: 658 ms
Please ensure you're using the latest port files with `git pull` and `vcpkg update`.
Then check for known issues at:
  https://github.com/microsoft/vcpkg/issues?q=is%3Aissue+is%3Aopen+in%3Atitle+zookeeper
You can submit a new issue at:
  https://github.com/microsoft/vcpkg/issues/new?title=%5Bzookeeper%5D%20build%20error%20on%20x64-windows&body=Copy%20issue%20body%20from%20D%3A%2FProjects%2FVS%2Fvcpkg%2Finstalled%2Fvcpkg%2Fissue_body.md

### Additional context

_No response_

--- a/ports/zookeeper/portfile.cmake
+++ b/ports/zookeeper/portfile.cmake
@@ -5,8 +5,9 @@ string(REGEX REPLACE "^([0-9]+[.][0-9]+[.][0-9]+)[.]([0-9]+)\$" "\\1-\\2" VERSIO
 vcpkg_download_distfile(
     zookeeper_src_archive
     URLS "https://dlcdn.apache.org/zookeeper/stable/apache-zookeeper-${VERSION}.tar.gz"
+         "https://archive.apache.org/dist/zookeeper/zookeeper-${VERSION}/zookeeper-${VERSION}.tar.gz"
     FILENAME "apache-zookeeper-${VERSION}.tar.gz"
-    SHA512 78d909c92b3709cc2112d1b8df9ef006f78a81ee0aa1b6b6400b8fea771ebaafc03cde497c6080e3fd924b75facb28420c4970885914e5dc9cd47cd761e96dd4
+    SHA512 61c05f6064797994dc25c42df35d67d2c3839fd59a496924852a4d78b492b06746c8eb5445edb63cbc0107ef2b8b31babf23488f96a52b00682cd2e9b61be339
 )
 vcpkg_extract_source_archive(
     SOURCE_PATH
@@ -22,8 +23,9 @@ file(COPY "${CURRENT_PORT_DIR}/unofficial-zookeeperConfig.cmake" DESTINATION "${
 vcpkg_download_distfile(
     zookeeper_bin_archive
     URLS "https://dlcdn.apache.org/zookeeper/stable/apache-zookeeper-${VERSION}-bin.tar.gz"
+         "https://archive.apache.org/dist/zookeeper/zookeeper-${VERSION}/zookeeper-${VERSION}-bin.tar.gz"
     FILENAME "apache-zookeeper-${VERSION}-bin.tar.gz"
-    SHA512 4d85d6f7644d5f36d9c4d65e78bd662ab35ebe1380d762c24c12b98af029027eee453437c9245dbdf2b9beb77cd6b690b69e26f91cf9d11b0a183a979c73fa43
+    SHA512 ab9bf90649df19d8fd8378f2e8d9159bc8528d8e4c166a93d9fa4a9c98e39ee9de0279cc9dc58cd6d593141c0a45576d0df9db47d143d63951598a43efdc0a30
 )
 vcpkg_extract_source_archive(
     zookeeper_jute_path

--- a/ports/zookeeper/vcpkg.json
+++ b/ports/zookeeper/vcpkg.json
@@ -1,6 +1,6 @@
 {
   "name": "zookeeper",
-  "version": "3.8.4",
+  "version": "3.8.5",
   "description": "ZooKeeper C bindings",
   "homepage": "https://github.com/apache/zookeeper",
   "license": "BSD-3-Clause",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -10873,7 +10873,7 @@
       "port-version": 0
     },
     "zookeeper": {
-      "baseline": "3.8.4",
+      "baseline": "3.8.5",
       "port-version": 0
     },
     "zopfli": {

--- a/versions/z-/zookeeper.json
+++ b/versions/z-/zookeeper.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "eaef57b80cbd6af354d909e4b1ec1f5a9b1603cb",
+      "version": "3.8.5",
+      "port-version": 0
+    },
+    {
       "git-tree": "4ccb6991bb5bf41c030df4c0e6b371e3d2fd6136",
       "version": "3.8.4",
       "port-version": 0

